submit-cron/templates/new_user_invitation.html

```diff
@@ -1,74 +1,86 @@
-        }
-        .link-box {
-            border-left: 4px solid #a6a6a6;
-            padding-left: 10px;
-            margin: 15px 0;
-        }
-        .cta-link {
-            color: #1a5a96; /* BC Gov link color */
-            font-size: 16px;
-            font-weight: bold;
-            text-decoration: none;
-        }
-        .cta-link:hover {
-            text-decoration: underline;
-        }
-        .footer {
-            font-size: 12px;
-            color: #666;
-            margin-top: 20px;
-        }
-    </style>
+  <meta charset="UTF-8">
+  <title>Invitation to collaborate on EPIC.submit</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      color: #003366;
+      background-color: #ffffff;
+      padding: 20px;
+    }
+    .container {
+      max-width: 600px;
+      margin: auto;
+      padding: 20px;
+    }
+    .logo {
+      text-align: left;
+      margin-bottom: 20px;
+    }
+    .logo img {
+      max-width: 220px;
+      height: auto;
+    }
+    .content {
+      font-size: 16px;
+      line-height: 1.6;
+      color: #333;
+      text-align: left;
+    }
+    .highlight-box {
+      background-color: #FAF9F8;
+      padding: 16px;
+      border-left: 4px solid #1a5a96;
+      margin: 20px 0;
+    }
+    .cta-link {
+      color: #1a5a96;
+      font-weight: bold;
+      text-decoration: none;
+    }
+    .cta-link:hover {
+      text-decoration: underline;
+    }
+    .footer {
+      font-size: 12px;
+      color: #666;
+      margin-top: 30px;
+      text-align: left;
+    }
+  </style>
 </head>
 <body>
-    <div class="container">
-        <div class="logo">
-            <img src={{ logo_url }}
-                 alt="BC Government Logo">
-        </div>
-        <div class="content">
-            <p>You have been invited to <strong>{{ invitation_action_text }} {{ project_name }} </strong> on EPIC.submit.</p>
-            <p>Click the link below and sign in with a <strong>Business BCeID</strong> or <strong>BC Services Card</strong> to access the project.</p>
+  <div class="container">
+    <div class="logo">
+      <img src="{{ logo_url }}" alt="EAO Logo" />
+    </div>
+
+    <div class="content">
+      <p>You have been invited to join <strong>{{ project_name }}</strong> on EPIC.submit.</p>
+
+      <p>Click the link below and sign in with your Business BCeID or BC Services Card to access the project.</p>
+
+      <div class="highlight-box">
+        <a href="{{ invitation_url }}" class="cta-link">{{ invitation_url }}</a>
+      </div>
+
+      <p>
+      If you don’t have a Business BCeID associated with this project, please contact the person in charge of the Business BCeID account for {{ certicate_holder_name }}.
+      </p>
 
-            <div class="link-box">
-                <a href="{{ invitation_url }}" class="cta-link">Join EPIC.submit</a>
-            </div>
+      <p>
+       For more information on how to use or set up a BC Services Card account, visit {{ bceid_link }}.
```

Comment:
For more information on how to use or set up a BC Services Card account, visit {{ bceid_link }}.????

Reply:
in the task it says to change the text to For more information on how to use or set up a BC Services Card account, visit [www.id.gov.bc.ca](https://www.id.gov.bc.ca/)